boards/common/esp8266/Kconfig

```diff
@@ -21,7 +21,7 @@ config BOARD_COMMON_ESP8266
 config MODULE_BOARDS_COMMON_ESP8266
     bool
     depends on TEST_KCONFIG
-    depends on HAS_ARCH_ESP8266
+    depends on BOARD_COMMON_ESP8266
```

Comment:
```
    depends on HAS_ARCH_ESP8266
```
was correct because `BOARD_COMMON_ESP8266` doesn't depend on `HAS_ARCH_ESP8266` if I'm right. My suggestion was to add
```
depends on BOARD_COMMON_ESP8266
```
as for ESP32 to have the dependencies defined by `BOARD_COMMON_ESP8266`. Or am I wrong?

Reply:
Hmm yeah. I re-added the dependency on the feature fc0c3d83c27d39b674f03aa0064b6d1d92c9c582